.travis.yml

```diff
@@ -8,6 +8,10 @@ addons:
       - libhiredis-dev
       - redis-server
       - libv8-dev
+      - libjq-dev
+      - libudunits2-dev
+      - libprotobuf-dev
+      - protobuf-compiler
```

Comment:
Also after now seeing the amount of installs required by `geojsonio` I'm wondering if there is a better package to use?

Reply:
All geo stuff is awful for dependencies - most of the involved depends here are probably for the shapefile processing rather than the geojson part. Nothing obviously jumps out as a better bet